src/pages/index.js

```diff
@@ -0,0 +1,361 @@
+import {
+  enableValidation,
+  validationConfig,
+  resetValidation,
+  disableBtn,
+} from "../scripts/validation.js";
+import "../pages/index.css";
+import Api from "../utils/api.js";
+import { setButtonText } from "../utils/helper.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "0461a93f-e4a4-4710-9671-530f1a6885c6",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([userInfo, cards]) => {
+    //Handle the user's information
+    // - set the src of the avatar image
+    document.querySelector(".profile__avatar").src = userInfo.avatar;
```

Comment:
`document.querySelector(".profile__avatar")` 

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine